packages/cyberchef.vm/cyberchef.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>cyberchef.vm</id>
+    <version>9.32.3</version>
+    <authors>GCHQ</authors>
+    <description>The Cyber Swiss Army Knife - a web app for encryption, encoding, compression, data analysis, and more.</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="googlechrome" />
```

Comment:
QUESTION: why are we using googlechrome vs firefox for example?

Reply:
Added here: https://github.com/mandiant/VM-Packages/issues/2#issuecomment-998685033

Reply:
CyberChef supports:
- Google Chrome 50+
- Mozilla Firefox 38+

So I picked Chrome since it also has good debugging tools. Since it's a dependency we'd need to pick one or the other, but realistically we probably don't need both. Thoughts?

Reply:
The dependency makes it more difficult to make it configurable. It was just an idea, let's keep it like it is by now.